File: test/fixtures/output/node/request/text-plain.js
var request = require("request");

request({
  "method": "POST",
  "url": "http://mockbin.com/har",
  "headers": {
    "content-type": "text/plain"
  },
  "body": "Hello World"
}, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: src/targets/csharp/httpclient/fixtures/text-plain.cs
var client = new HttpClient();
var request = new HttpRequestMessage
{
    Method = HttpMethod.Post,
    RequestUri = new Uri("http://mockbin.com/har"),
    Content = new StringContent("Hello World")
    {
        Headers =
        {
            ContentType = new MediaTypeHeaderValue("text/plain")
        }
    }
};
using (var response = await client.SendAsync(request))
{
    response.EnsureSuccessStatusCode();
    var body = await response.Content.ReadAsStringAsync();
    Console.WriteLine(body);
}
